File: BackendAPI/src/main/java/com/aguare/backendapi/Entitys/Product.java
package com.aguare.backendapi.Entitys;

import jakarta.persistence.*;
import lombok.Data;

import java.io.Serializable;

@Entity
@Data
@Table(name = "product")
public class Product implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long idProduct;
    private String name_p;
    private Double price;
    private String description;
}

File: FrontApp/src/app/Entitys/Product.ts
export class Product {
  idProduct: number;
  name_p: string;
  price: number;
  description: string;

  constructor(
    idProduct: number,
    name_p: string,
    price: number,
    description: string
  ) {
    this.idProduct = idProduct;
    this.name_p = name_p;
    this.price = price;
    this.description = description;
  }
}
